test/unit/provision/synchronousButtonProvision.json

```diff
@@ -28,6 +28,18 @@
           "name": "op_result",
           "type": "string"
         }
+      ],
+      "static_attributes": [
+        {
+          "name": "interaction_type",
+          "type": "string",
+          "value": "synchronous"
+        },
+        {
+          "name": "service_id",
+          "type": "string",
+          "value": "smartcity"
+        }
```

Comment:
Other possible static attributes are: internal_id, ccid, imei, imsi, aux_interaction_type and geolocation, aren't they?